longruntests/longruntests_suite_test.go

```diff
@@ -0,0 +1,152 @@
+package longruntests_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/deis/workflow-e2e/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/onsi/ginkgo/reporters"
+)
+
+func getDir() string {
```

Comment:
I'd still like to see if it would be possible to have all the code duplicated here (`getDir()`, `init()`, `getRandAppName()`, `TestTests`, `var` listing, etc.) exist in only one file where possible (probably `tests_suite_test.go` or the new `utils`).